README.md

```diff
@@ -1,9 +1,10 @@
 # websec-check
-repo and tooling for https://wiki.mozilla.org/Security/FirefoxOperations#Security_Checklist
+
+repo and tooling for [wikimo checklist](https://wiki.mozilla.org/Security/FirefoxOperations#Security_Checklist)
 
 Markdown issue template:
 
-```
+```markdown
```

Comment:
:+1:

Reply:
I have to thank vscode's markdown linter for letting me know that was possible 😁